src/style.css

```diff
@@ -1,3 +1,84 @@
+:root {
+  --header-bg: #fff;
+  --header-height: 60px;
+  --nav-link-color: #000;
+  --nav-link-hover-color: #00acdc;
+  --nav-font-size: 12px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: var(--header-bg);
+  width: 100%;
+  padding: 0 50px;
+}
+
+.logo_link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav_list {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  list-style: none;
+  flex-direction: row;
+  padding: 0;
+  margin: 0;
+}
+
+.nav_list .nav_item:not(:last-of-type) {
+  margin-right: 20px;
+}
+
+.nav_link {
+  display: flex;
+  position: relative;
+  justify-content: center;
+  align-items: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: var(--nav-link-color);
+  font-family: Roboto, sans-serif;
+  font-size: var(--nav-font-size);
+  font-weight: 500;
+  height: var(--header-height);
+  line-height: 14.6px;
```

Comment:
The `line-height` of 14.6px might not be optimal for the `font-size` set by `--nav-font-size`. Consider adjusting the `line-height` to ensure proper vertical alignment and readability of the text.